src/cli.ts

```diff
@@ -21,15 +27,39 @@ async function main() {
 
       Compile all .abc files in src/ to build/
       $ template-file stuff.json 'src/**/*.abc' build/
-  `
-    // { importMeta: import.meta }
+  `);
+
+  console.log(usageString);
```

Comment:
console log here

Reply:
That's intentional. It outputs the usage text (e.g. `template-file --help`)